<?xml version="1.0"?>
<launch>
   <include file="$(find ass2)/launch/gazebo.launch" />
   <!-- <include file="$(find ass2)/launch/manipulator.launch" /> -->
   <include file="$(find ass2)/launch/joint_state_publisher.launch" />
   <include file="$(find ass2)/launch/robot_state_publisher.launch" />
   <include file="$(find ass2)/launch/rviz.launch" />
   <include file="$(find ass2)/launch/amcl.launch">
      <arg name="scan_topic" value="/manmaru/laser_scan" />
   </include>
   <!-- <include file="$(find ass2)/launch/gmapping.launch">
      <arg name="scan_topic" value="/manmaru/laser_scan" />
   </include> -->
   <include file="$(find ass2)/launch/map_server.launch" />
   <include file="$(find ass2)/launch/move_base.launch">
      <arg name="cmd_vel_topic" value="/manmaru/cmd_vel" />
      <arg name="odom_topic" value="/manmaru/odom" />
   </include>
   <include file="$(find ass2)/launch/face_detector.launch">
      <arg name="image_raw_topic" value="/manmaru/camera_front/image_raw" />
      <arg name="face_detection_topic" value="/manmaru/object_detection" />
   </include>
</launch>